docs/quickstarts/nextjs.mdx

```diff
@@ -5,32 +5,8 @@ description: Add authentication and user management to your Next.js app with Cle
 
 # Next.js Quickstart
 
-<TutorialHero
-  framework="nextjs"
-  beforeYouStart={[
-    {
-      title: "Set up a Clerk application",
-      link: "https://clerk.com/docs/quickstarts/setup-clerk",
-    },
-    {
-      title: "Create a Next.js application",
-      link: "https://nextjs.org/docs/getting-started/installation",
-    },
-  ]}
->
-
-- Install `@clerk/nextjs`
-- Set up your environment variables
-- Wrap your Next.js app in `<ClerkProvider />`
-- Limit access to authenticated users
-- Specify which routes are publicly accessible
-- Enable users to manage their session with `<UserButton />`
-
-</TutorialHero>
-
-<Callout type='info' emoji='💡'>
-  The next major version of the Clerk Next.js SDK is coming soon! If you're just starting a project, consider [trying out the Core 2 beta](https://beta.clerk.com/docs/quickstarts/nextjs) instead.
-</Callout>
+Protect your Next.js application with Clerk auth in ten minutes. This guide will show you how to install Clerk, update your middleware and embed our `<UserButton />` component. 

```

Comment:
What's the idea behind specifically adding "ten minutes"? I know it can be done in less than that and intuitively, it feels more harmful than helpful to advertise an exact number for setting up Clerk 😱

And if this suggestion is too "wordy", then at the least, let's remove the use of "our".

```suggestion
Protect your Next.js application with Clerk auth in ten minutes. This guide will show you how to install Clerk, add middleware to protect specific routes, and embed the `<UserButton />` component for account management.

```